Calculator.Console/Program.cs

```diff
@@ -6,8 +6,48 @@ class Program
     {
         static void Main(string[] args)
         {
-            System.Console.WriteLine("Hello World!");
-            var name = System.Console.ReadLine();
+            int a = 0, b = 0;
```

Comment:
Testing this code is still possible.
Think about what dependencies you have in this code, and try to abstract those dependencies.
(Hint: System.Console is an ugly dependency ... if you wouldn't have it you would be able to test your code very easy)